### ISSUE TITLE
advice

### ISSUE THREAD

**Issue by UchihaSR** [adam-mcdaniel/resume#1]
sorry that I am bothering you here. I know this is not the place to ask personal questions. it has been months that I am trying to learn to code. I can grasp a small codebase but not the large one. never made any small project on my own. i want to try and build something. i am seeking your advice on how should i do it?. even at this younger age you did well.

**Comment by adam-mcdaniel** [comment]
Hello, @UchihaSR. I first learned to program using Python several years ago, and then branched out to several other languages as I became comfortable with the major concepts that most languages share. If you go back far enough in my GitHub profile, you'll see that I started by writing a few terrible games that were just fun to make. Additionally, it's REALLY difficult to read code if you haven't been programming for long. I always have trouble reading code for a program I'm not familiar with. I highly recommend creating your own projects before trying to understand other peoples'.

If you want to make a project, start small. Maybe write a program that generates ASCII art trees, or a small game that runs in the terminal.

I started learning with [this particular playlist](https://youtube.com/playlist?list=PLQVvvaa0QuDdLkP8MrOXLe_rKuf6r80KO)
